catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/suggestion/LatLonCoordinateProcessor.java

```diff
@@ -58,7 +56,7 @@ private LiteralSuggestion getLatLonSuggestions(String query) {
     List<LatLon> latLonList = getLatLonList(numbers);
     String name =
         "Lat/Lon: " + latLonList.stream().map(LatLon::toString).collect(Collectors.joining(", "));
-    return new LiteralSuggestion("LITERAL", name, latLonList);
+    return new LiteralSuggestion("LITERAL1", name, latLonList);
```

Comment:
❓ What does `LITERAL1` signify?

Reply:
Typically a returned JSON payload from `/geofeature/suggestion?q=query` would look like this:
```
[
  {
    "id": "32100003-65e84e8c-b8fe0769-8b6446e7",
    "name": "Tempe, AZ"
  },
  {
    ...
  }
]
```
We provide a special ID that has significance to the UI but is still unique to prevent errors in the React component. An ID that starts with `LITERAL` means the geo is already on the suggestion and can be used immediately. The constant is just a counter for the literal suggestion. Since there are currently 2 at most, 1 is the Lat/Lon suggestion and 2 is the MGRS suggestion. 
```
{
  "id": "LITERAL1",
  "name": "Lat/Lon: (1, 2)",
  "geo": { ... }
}
```
I can add javadoc for this.

Reply:
My hang up with this is the 1 and 2. I don't believe you're currently using these constants, but the way I see it, if we were to need to distinguish between literal types somewhere down the line, 1 and 2 don't carry much significance to someone without context

Reply:
So if `LITERAL1 == LAT/LONG` and `LITERAL2 == MGRS` then change `id` to `type` and be specific. I don't see a reason to add an ambiguous mapping